components/confluence/actions/get-page-by-id/get-page-by-id.mjs

```diff
@@ -19,6 +21,47 @@ export default {
         "pageId",
       ],
     },
+    bodyFormat: {
+      type: "string",
+      label: "Body Format",
+      description: "The content format types to be returned in the `body` field of the response. If available, the representation will be available under a response field of the same name under the `body` field.",
+      optional: true,
+      options: BODY_FORMAT_FULL_OPTIONS,
+    },
```

Comment:
Is there a reason this doesn't use the `bodyFormat` prop from the app file?

Reply:
No, thanks for making the update, it should've reused hte propDefinition and overridden only the options